tawazi/_errors.py

```diff
@@ -35,6 +35,12 @@ class TawaziUsageError(TawaziBaseException):
     pass
 
 
+class TawaziTimeoutError(TawaziBaseException):
```

Comment:
shouldn't it also inherit from the base `TimeoutError` ?

Reply:
I agree